packages/twenty-server/src/database/typeorm-seeds/core/demo/feature-flags.ts

```diff
@@ -1,26 +1,17 @@
 import { DataSource } from 'typeorm';
 
-import { FeatureFlagKeys } from 'src/core/feature-flag/feature-flag.entity';
-
 const tableName = 'featureFlag';
 
 export const seedFeatureFlags = async (
   workspaceDataSource: DataSource,
   schemaName: string,
-  workspaceId: string,
 ) => {
   await workspaceDataSource
     .createQueryBuilder()
     .insert()
     .into(`${schemaName}.${tableName}`, ['key', 'workspaceId', 'value'])
     .orIgnore()
-    .values([
-      {
-        key: FeatureFlagKeys.IsNewRecordBoardEnabled,
-        workspaceId: workspaceId,
-        value: false,
-      },
-    ])
+    .values([])
```

Comment:
Do we want to comment this whole block then? I'm not sure if this will execute a query or not 🤔